.github/workflows/backport-docs.yml

```diff
@@ -21,6 +21,7 @@ jobs:
           BACKPORT_TARGET_TEMPLATE: "stable-{{.target}}"
           GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
       - name: Backport changes labeled website to latest release branch
+        if: always()
```

Comment:
In the docs (https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-not-requiring-dependent-jobs-to-be-successful) this is surrounded with `{{` - do you happen to know whether those are necessary and if not why?

Reply:
@mladlow I had to google this... Found the answer [here](https://github.community/t/multiple-if-conditional-with-always-is-always-true/128840/3), which ultimately pointed right back to the docs 😂 
- https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepsif

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. For more information, see "[Expressions](https://docs.github.com/en/actions/learn-github-actions/expressions)."

Reply:
Thank you!